changelog/2.072.0_pre.dd

```diff
@@ -730,6 +732,16 @@ $(BUGSTITLE Library Changes,
                             // now it works as expected
     -------
     )
+
+    $(LI $(LNAME2 drt-oncycle, New druntime switch `--DRT-oncycle` allows
+        specifying what to do on cycle detection in modules.)
+        $(P When module cycles are detected, the default behavior is to print the cycle, and abort execution. However, in many cases, the cycles are not harmful. With the new `--DRT-oncycle` switch, you can effect a different behavior when cycles are detected:
+        $(OL
```

Comment:
Close the P before opening the OL. Also, a definition list (DL) may be a better fit.